docs/apache-airflow-providers-microsoft-azure/connections/azure.rst

```diff
@@ -53,10 +56,12 @@ Configuring the Connection
 Login (optional)
     Specify the ``client_id`` used for the initial connection.
     This is only needed for *token credentials* authentication mechanism.
+    It can be left out to fall back on ``DefaultAzureCredential``.
```

Comment:
```suggestion
    It can be left out to fall back on DefaultAzureCredential_.
```

We can pretty easily add links to all occurrences like this. Not sure if we want to do this though.

Reply:
That sounds like a great idea!